nightscout-api/src/main/java/com/dongtronic/nightscout/Nightscout.kt

```diff
@@ -108,14 +109,16 @@ class Nightscout(baseUrl: String, token: String? = null) : Closeable {
             val top = ranges.path("bgTargetTop").asInt()
             val high = ranges.path("bgHigh").asInt()
 
-            dto.title = title
-            dto.low = low
-            dto.bottom = bottom
-            dto.top = top
-            dto.high = high
-            dto.units = units
+            val builder = dto.newBuilder()
```

Comment:
Is it possible to chain all the setters here? Like `newBuilder().title(title).low(low).bottom(bottom)` etc